example/build.gradle.kts

```diff
@@ -1,5 +1,5 @@
 plugins {
-    java
+    id("mcprotocollib.base-conventions")
```

Comment:
I don't really think this is necessary since it'd be nice to keep the example as simple as possible, and the base conventions required anyway